src/Logbook.js

```diff
@@ -1,23 +1,60 @@
 import React, { useState } from 'react'
 import { useForm } from 'react-hook-form'
+import { storage } from './firebase'
 import uuid from 'react-uuid'
 import styled from 'styled-components'
 import { postLogs } from './services'
 
 export default function LogBook() {
-  const [dives, setDives] = useState([])
-  const { register, handleSubmit } = useForm({ defaultValues: { id: uuid() } })
+  const [image, setImage] = useState(null)
+  const [url, setUrl] = useState()
+  const { register, handleSubmit } = useForm({
+    defaultValues: { id: uuid(), img: url },
+  })
 
-  const onSubmit = data => {
-    setDives([...dives, { data }])
-    postLogs(data)
+  const handleUpload = () => {
```

Comment:
Geil, das sieht sehr gut aus und intuitiv. Cool, dass das so klappt. Wie ist denn Storage in der Firebase.js genau definiert? Brauchst du das Package `react-firebase-file-uploader` eigentlich noch oder löst du es mit dem, was Firebase liefert?